lib/wings/active_fedora_converter.rb

```diff
@@ -190,17 +188,10 @@ def assign_association_target(af_object, pcdm_file)
       def normal_attributes
         attributes.each_with_object({}) do |(attr, value), hash|
           property = active_fedora_class.properties[attr.to_s]
-          # This handles some cases where the attributes do not directly map to an RDF property value
-          hash[attr] = value
-          next if property.nil?
-          hash[attr] = Array.wrap(value) if property.multiple?
+          hash[attr] = property&.multiple? ? Array.wrap(value) : value
```

Comment:
i'm hoping this refactor in-particular is clarifying. previously i was wondering which "some cases" needed this special handling; and why, if this logic is narrowly targeted, it applies to all attributes. after looking at it closely, what we actually do is cast `value` to `Array` if `property.multiple?` is `true`, otherwise set the `value` as-is.

the special case is where `property` is `nil`. for me, this seems like a perfect use of `&.`. we can just treat `nil` properties as non-multiple, and not interrupt the flow of logic for a special check.